domain/consensus/processes/ghostdagmanager/compare.go

```diff
@@ -45,26 +46,13 @@ func (gm *ghostdagManager) ChooseSelectedParent(blockHashA *externalapi.DomainHa
 	blockABlueScore := blockAGHOSTDAGData.BlueScore
 	blockBBlueScore := blockBGHOSTDAGData.BlueScore
 	if blockABlueScore == blockBBlueScore {
-		if hashesLess(blockHashA, blockHashB) {
-			return blockHashB, nil
+		if hashes.Less(blockHashA, blockHashB) {
+			return blockHashA, nil
 		}
-		return blockHashA, nil
+		return blockHashB, nil
```

Comment:
I believe this is a consensus change and will break all the ghostDAG tests

Reply:
Before we took the higher block and here we take the lower:
https://github.com/kaspanet/kaspad/blob/eaf911722584b8ce732306c4250b220b51499d0f/domain/blockdag/blockset.go#L108-L109